src/client/ui/EditorContainer.js

```diff
@@ -70,7 +72,6 @@ class EditorContainer extends Component {
         },
         properties: {
           component: PropertiesPanelContainer,
-          props: { openFileDialog: this.openFileDialog },
```

Comment:
The material save/load buttons are broken now. You removed `openFileDialog` here but you didn't make the corresponding changes in `PropertiesPanelContainer`.

Reply:
This also broke the file dialog in FileInput

Reply:
Ah, right I forgot about that. Thanks!